pos_order_mgmt/static/src/js/models.js

```diff
@@ -27,4 +27,5 @@ odoo.define('pos_order_mgmt.models', function (require) {
             return res;
         },
     });
+
```

Comment:
Why adding this meaningless empty line?

Reply:
That's a question I don't have an answer for :+1: 
Removed it!